tests/ClientTest.php

```diff
@@ -219,19 +221,22 @@ public function testSendChecksBeforeSendOption(): void
     /**
      * @dataProvider sampleRateAbsoluteDataProvider
      */
-    public function testSampleRateAbsolute($options): void
+    public function testSampleRateAbsolute(float $sampleRate): void
     {
         $httpClient = new MockClient();
 
-        $client = ClientBuilder::create($options)
+        $options = new Options(['dsn' => 'http://public:secret@example.com/1']);
```

Comment:
Instead of instantiating the `Options` object I would prefer to stick with the plain `array` of options. Also I would check if we really need a valid HTTP client or we can just set a mocked transport to assert that the `sample_rate` option works